map_model/src/objects/building.rs

```diff
@@ -227,110 +227,193 @@ fn sidewalk_to_bike(sidewalk_pos: Position, map: &Map) -> Option<(Position, Posi
 /// Businesses are categorized into one of these types.
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
 pub enum AmenityType {
-    Groceries,
-    Food,
     Bar,
-    Medical,
-    Religious,
-    Education,
+    Beauty,
+    BikeStore,
+    BikeParkRent,
+    CarRepair,
+    CarShare,
+    Childcare,
+    ConvenienceStore,
+    Culture,
     Financial,
+    Food,
+    Laundry,
+    Medical,
+    Pet,
     PostOffice,
-    Culture,
-    Childcare,
+    Religious,
+    School,
     Shopping,
+    Supermarket,
+    University,
 }
 
 impl AmenityType {
     fn types(self) -> Vec<&'static str> {
-        // TODO: create categories for:
-        // hairdresser beauty chemist
-        // car_repair
-        // laundry
-
         match self {
-            AmenityType::Groceries => vec!["convenience", "supermarket"],
-            // TODO Sort
+            AmenityType::Bar => vec!["bar", "lounge", "pub", "nightclub", "biergarten"],
+            AmenityType::Beauty => vec!["hairdresser", "beauty", "chemist", "cosmetics"],
+            AmenityType::BikeParkRent => vec!["bicycle_parking", "bicycle_rental"],
```

Comment:
`amenity=bicycle_parking` is often defined on individual nodes outside of buildings to tag bike racks. With this included here, right now it'll pick up bike parking tagged inside buildings. I think that's totally fine, just wanted to point out that this might be a little unexpected.

Reply:
Yes good point, I kind of forgot this is for buildings. I do think things like bike parking, car charging, etc could be useful beyond just the building categories looking forward but certainly plenty to work with using buildings categorized :) I will just remove that whole category I think @michaelkirk also commented on it.